pymc_marketing/mmm/lift_test.py

```diff
@@ -63,8 +62,19 @@ def lift_test_indices(df_lift_test: pd.DataFrame, model: pm.Model) -> Indices:
     """
 
     columns = df_lift_test.columns.tolist()
+
+    def _ensure_datetime_type_in_array(x: np.ndarray) -> np.ndarray:
```

Comment:
can you add a basic test for this one ?

Reply:
Added tests for the dates.
Went for slightly different solution and added comment instead
Let me know what you think